src/components/EventMerge/index.js

```diff
@@ -13,18 +13,24 @@ governing permissions and limitations under the License.
 import { uuid } from "../../utils";
 import { callback } from "../../utils/validation";
 
+const generateEventMergeIdResult = () => {
+  return {
```

Comment:
WHAT!!???? Now I have to do `result.eventMergeId`?? 🤣

Reply:
:rage4: